common/src/main/java/com/wynntils/features/inventory/ItemLockFeature.java

```diff
@@ -86,13 +90,18 @@ public void onInventoryClickEvent(ContainerClickEvent event) {
             return;
         }
 
-        // We want to allow opening emerald pouch even if locked
-        // Right click is opening pouch, left click is picking it up.
-        // We want to allow opening pouch even if locked, but not picking it up.
-        if (allowClickOnEmeraldPouchInBlockingMode.get()
-                && event.getMouseButton() == GLFW.GLFW_MOUSE_BUTTON_RIGHT
-                && Models.Emerald.isEmeraldPouch(slotOptional.get().getItem())) {
-            return;
+        // We want to allow opening emerald pouches and deleting potions even if locked
+        // Right click is used to perform these actions, left click picks them up
+        // So only allow right click actions
+        if (event.getMouseButton() == GLFW.GLFW_MOUSE_BUTTON_RIGHT) {
+            if (allowClickOnEmeraldPouchInBlockingMode.get()
+                    && Models.Emerald.isEmeraldPouch(slotOptional.get().getItem())) {
+                return;
+            } else if (allowClickOnMultiHealthPotionsInBlockingMode.get()
+                    && Models.Item.asWynnItem(slotOptional.get().getItem(), MultiHealthPotionItem.class)
+                            .isPresent()) {
+                return;
+            }
```

Comment:
```suggestion
            if (allowClickOnEmeraldPouchInBlockingMode.get()
                    && Models.Emerald.isEmeraldPouch(slotOptional.get().getItem())) {
                return;
            }
            
             if (allowClickOnMultiHealthPotionsInBlockingMode.get()
                    && Models.Item.asWynnItem(slotOptional.get().getItem(), MultiHealthPotionItem.class)
                            .isPresent()) {
                return;
            }
```

Should help with the indentation a tiny bit.